### PR TITLE
Integration test library functions for listing nodes and changing nodes (for LBSH tests)

### PR DESCRIPTION
**Pull request by cyli**
So we can start working on the load balancer self healing tests.

### CODE REVIEW

**Comment by cyli** [comment]
jenkins please run integration tests

**Comment by cyli** [comment]
jenkins please run integration tests

File: otter/integration/lib/cloud_load_balancer.py [hunk 1]
```diff
@@ -178,6 +178,41 @@ def delete(self, rcs, success_codes=None):
                 [202, 404] if success_codes is None else success_codes)
         ).addCallback(lambda _: rcs)
 
+    def list_nodes(self, rcs):
+        """
+        Get all the nodes on the load balancer.
+        But this function just returns the list of dictionaries.
```
Comment (radix) [inline otter/integration/lib/cloud_load_balancer.py]: Not sure exactly what this line is saying (especially considering the "But"). Is it describing anything that's not already covered by the `:return:`?

**Comment by radix** [comment]
LGTM. you can ignore my one docstring night and just merge if you want :)

File: otter/integration/lib/cloud_load_balancer.py (hunk 1 above)
Comment (cyli) [inline otter/integration/lib/cloud_load_balancer.py]: Oops!  Sorry, this was leftover from a previous edit.  Fixing - thanks for spotting that!

**Pull request merged**